c7n/resources/asg.py

```diff
@@ -647,12 +647,12 @@ class VpcIdFilter(ValueFilter):
 
     .. code-block:: yaml
 
-            policies:
-              - name: asg-vpc-xyz
-                resource: asg
-                filters:
-                  - type: vpc-id
-                    value: vpc-12ab34cd
+        policies:
+          - name: asg-vpc-xyz
+            resource: asg
+            filters:
+              - type: vpc-id
+                value: vpc-12ab34cd
```

Comment:
That’s not a valid filter

Reply:
how so?

Reply:
oh.. sorry i guess it is, it just seems odd to have a dedicated id filter when its commonly referenced via value filter as an attribute, this is one thats worth deprecating for a `vpc` filter that can look at additional vpc attributes (tags, etc).

Reply:
yeah, agreed. it actually says as much in the docstring.

Reply:
actually, the vpc-id does not show up in the `describe_autoscaling_groups` generic results.. the `VPCZoneIdentifier` gives a subnet id which is what the filter keys on to find the vpc.

Reply:
right, but doesn't really change that this should be noted in the docs as deprecated, and we should have a vpc filter here that can more usefully filter asgs based on other vpc attributes.

Reply:
@kapilt wouldn't we need to have an alternative before we denote the functionality as deprecated?  I don't think we have the vpc filter functionality yet.  This PR is just updating incorrect documentation.

Reply:
thats fair, in this context though its not even correcting (re vpc-id), its just reformatting it.

Reply:
in fact this whole pr is an exercise in reformatting not correctness.